atomix/cluster/src/main/java/io/atomix/raft/storage/RaftStorage.java

```diff
@@ -187,7 +187,7 @@ public boolean dynamicCompaction() {
   /**
    * Returns the percentage of disk space that must be available before log compaction is forced.
```

Comment:
This needs then probably also be changed
```suggestion
   * Returns the amount of disk space that must be available before log compaction is forced.
```